baseball/src/App.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import "./css/App.scss";
+
+class App extends React.Component {
```

Comment:
good job building out your basic react app and it's functionalities to be tested